upgrade/rendering-changes/4-3-0.md

```diff
@@ -21,6 +21,53 @@ position: 1
 
 The default icon type in the Telerik UI for Blazor suite changes from `FontIcon` to `SvgIcon`. This marks the next milestone in a series of improvements related to the [Content Security Policy (CSP)]({%slug troubleshooting-csp%}).
 
+### Switching to SVG icons
+
+When upgrading your application to 4.3.0 you might experience two possible scenarios related to the new default icon type:
```

Comment:
Specify when the customer will experience the scenarios, so that there is no wondering and doubts.